js/modules/books.js

```diff
@@ -0,0 +1,70 @@
+// create a class for book collection
+export default class Books {
+  constructor() {
+    // eslint-disable-next-line no-unused-expressions
+    this.list;
+  }
+
+  // get books from local storage
+  // eslint-disable-next-line class-methods-use-this
+  get list() {
+    return localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
+  }
+
+  // render books
+  renderBooks() {
+    // Get DOM elements'
+    const bookList = document.querySelector('.books__list');
+
+    // get the books from the books array from localStorage
+    const booksCollection = this.list;
+
+    // check the length of the books array
+    if (booksCollection.length === 0) {
+      // clear the list
+      bookList.innerHTML = '';
+
+      // update the DOM with no books message
+      bookList.innerHTML = '<p class="books__list-item">No books to show</p>';
+    } else {
+      // loop through the books array
+      const bookshtml = booksCollection.map((book) => `
+
+      <div class="books__list-item">
+        <p class="books__list-book">"${book.title}" by ${book.author}</p>
+        <button class="books__list-item-delete" data-id="${book.id}">Delete</button>
+      </div>
+    `).join('');
+
+      // clear the list
+      bookList.innerHTML = '';
+
+      // update the DOM with the new list
+      bookList.insertAdjacentHTML('afterbegin', bookshtml);
+    }
+  }
+
+  // add a book function
+  // eslint-disable-next-line class-methods-use-this
+  addBook(book) {
+    // get the books from the books array from localStorage
+    const booksCollection = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
+    booksCollection.push(book);
+
+    // update the books array in localStorage
+    localStorage.setItem('books', JSON.stringify(booksCollection));
```

Comment:
- **[OPTIONAL]** Please review the add function when I tried to add a book it didn't work and also review the delete function just in case for a better UX.

![image](https://user-images.githubusercontent.com/58642949/220412219-3378fae7-595a-47d4-b523-6e53fd018d99.png)